worlds/alttp/Rom.py

```diff
@@ -279,26 +279,26 @@ def apply_random_sprite_on_event(rom: LocalRom, sprite, local_random, allow_rand
                 rom.write_bytes(0x307078 + (i * 0x8000), sprite.glove_palette)
 
 
-def patch_enemizer(world, player: int, rom: LocalRom, enemizercli, output_directory):
+def patch_enemizer(multiworld, world, player: int, rom: LocalRom, enemizercli, output_directory):
```

Comment:
either `world = multiworld.worlds[player]` at the top or just pass world.

In general, I think the nicer design is to lay out functions as if they were members of the one alttp world (where that makes sense). I.e. make it work the same as `generate_output`.

Reply:
this should be fixed now